opentech/templates/base.html

```diff
@@ -120,9 +120,9 @@
                 </section>
 
                 <div class="header__button-container">
-                    <a href="#" class="button button--transparent button--contains-icons">
+                    <a href="{% url 'users:login' %}" class="button button--transparent button--contains-icons">
                         <svg class="icon icon--person"><use xlink:href="#person-icon"></use></svg>
-                        Log in
```

Comment:
could use wire the logout button to `{% url 'users:logout' %}` since that exists as well

Reply:
For the public site, no, we wont be reasoning about the User's authentication for now. When we are working through the dashboard header we will add a logout.